tests/test_views_admin.py

```diff
@@ -0,0 +1,242 @@
+from pyramid.config import Configurator
+import unittest
+
+from kinto.core import events
+from kinto.core.storage import exceptions
+from kinto.core.testing import get_user_headers
+from kinto.core import utils
+from kinto.views.admin import get_parent_uri
+
+from .support import BaseWebTest, MINIMALIST_BUCKET, MINIMALIST_COLLECTION, MINIMALIST_RECORD
+
+
+class DeleteUserDataTest(BaseWebTest, unittest.TestCase):
+    doomed_bucket_url = "/buckets/user_to_delete"
+    doomed_collection_url = doomed_bucket_url + "/collections/collection_to_delete"
+    doomed_bucket_collection_url = doomed_bucket_url + "/collections/collection"
```

Comment:
I think it would help if you would briefly explain (here or in setup) what you mean by doomed and safe :)